ui/.yarnrc

```diff
@@ -0,0 +1 @@
+--install.no-lockfile true
```

Comment:
I spent some time trying to fix the package-lock.json hash churn. When I was researching, some people had success by simply using yarn instead of npm. I'm inclined to turn this on to see if, as I suspect, yarn's lockfile will not have the same issue.

Reply:
Its going to depend on whether `yarn` logs an entry for OS-specific or otherwise "optional" dependencies; I'm 99% sure that 99% of the churn from `package-lock` was `fsevents` being repeatedly added and removed.  Either way, we can turn it on and see, I'm ok with that.

Also @calbach you'll be happy to see that I added some flags to the test / build runs that suppress the progress bar spam.

Reply:
The churn I saw constantly was npm flipping between SHA512 and SHA1 hashes. This caused hundreds of lines of change in package-lock.json where the change was only the hashing algorithm used, not the actual version of the package. I'm hoping that yarn is simply a better implementation and won't do that.